robotiq_description/urdf/robotiq_gripper.ros2_control.xacro

```diff
@@ -1,21 +1,106 @@
 <?xml version="1.0"?>
 <robot xmlns:xacro="http://www.ros.org/wiki/xacro">
 
-    <xacro:macro name="robotiq_gripper_ros2_control" params="name com_port">
-        <ros2_control name="${name}" type="actuator">
+    <xacro:macro name="robotiq_gripper_ros2_control" params="
+        name
+        prefix
+        sim_ignition:=false
```

Comment:
Should we just bite the bullet and call this `sim_gazebo` as that is the way forward?

Reply:
I think this is going to get confusing. For example the [UR robot has both](https://github.com/UniversalRobots/Universal_Robots_ROS2_Description/blob/ros2/urdf/ur_macro.xacro#L71-L72).